perllib/FixMyStreet/App/Controller/Waste/Bulky.pm

```diff
@@ -311,11 +311,12 @@ sub add_cancellation_report : Private {
     $c->cobrand->call_hook( "waste_munge_bulky_cancellation_data", \%data );
 
     if ($c->cobrand->bulky_cancel_by_update) {
-        $collection_report->add_to_comments({
+        my $comment = $collection_report->add_to_comments({
             text => 'Booking cancelled by customer',
             user => $collection_report->user,
             extra => { bulky_cancellation => 1 },
         });
+        $collection_report->cancel_update_alert($comment->id);
```

Comment:
Whilst we want to cancel the payment confirmed one (because they'll have got/get a "report logged" email for the same event when it was sent to Echo), I don't think we want to cancel this one, because we want the user to get an email about the cancellation. (It's possible we'll get an update from Echo and then send two emails, but if that happens in testing we can then get that down to one, would rather start with too many.)